extension/src/dvcYaml/index.ts

```diff
@@ -0,0 +1,130 @@
+import { join } from 'path'
+import {
```

Comment:
Lots of changes here, but the plan is to move this code to the dvcYaml module, like it was done for all the other modules we have.